internal/impl/devbox.go

```diff
@@ -726,7 +727,45 @@ func (d *Devbox) installNixProfile() (err error) {
 		}
 
 		cmd.Env = nix.DefaultEnv()
-		_, err = cmd.Output()
+
+		// Get a pipe to read from standard out
+		r, _ := cmd.StdoutPipe()
+
+		// Use the same pipe for standard error
+		cmd.Stderr = cmd.Stdout
```

Comment:
This was from example I copy pasted (and probably correct) but just wanted to call out the previously we used `cmd.Output()` and not `CombinedOutput` so this changes it a bit.

Reply:
true, but we were ignoring the output returned from `cmd.Output()`, so I think this is okay for now.